src/sanitizer.js

```diff
@@ -58,6 +58,9 @@ const WHITELISTED_FORMAT_TAGS = [
 const BLACKLISTED_ATTR_VALUES = [
   /*eslint no-script-url: 0*/ 'javascript:',
   /*eslint no-script-url: 0*/ 'vbscript:',
+  /*eslint no-script-url: 0*/ 'data:',
+  /*eslint no-script-url: 0*/ '<script',
```

Comment:
Is it necessary? Attribute values will be properly encoded, so there's no chance to actually insert the angle bracket into the final HTML.